TestPlaybooks/script-VerifyTableToMarkDown.yml

```diff
@@ -32,6 +36,16 @@ script: |-
       throw "Simple Data failed to be converted to RIGHT Human Redable fotmat. Got: " + h;
   }
 
+  h = tableToMarkdown("Simple Array", JSON.parse(simpleArray), 'Array');
```

Comment:
why you define simpleArray as a string and then `JSON.parse` it? Why not define it as array from the beginning

Reply:
I was being consistent with the `VerifyTableToMarkdown` script, I assume there was a reason for it

Reply:
I don't see a valid reason reading the script. Please change this in all the places in the script to be object